src/widgets/guestStats/ui/GuestStatsTablesTitle/GuestStatsTablesTitle.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, Spin, Typography } from 'antd';
+import { FC } from 'react';
+import css from './GuestStatsTablesTitle.module.scss';
+
+const { Text } = Typography;
+
+interface Props {
+  marginBottom?: string;
+  isLoading?: boolean;
+  title: string;
+}
+
+export const GuestStatsTablesTitle: FC<Props> = ({ marginBottom, isLoading, title }) => {
+  return (
+    <Flex justify={'space-between'} align={'center'}>
+      <Text
+        strong
+        className={css.guestTableTitle}
+        style={{ marginBottom: marginBottom ? `${marginBottom}` : 0 }}
```

Comment:
Если  пропсы опциональные, лучше задавать сразу дефолтные значения
```suggestion
export const GuestStatsTablesTitle: FC<Props> = ({ marginBottom = '0', isLoading = true, title }) => {
  return (
    <Flex justify={'space-between'} align={'center'}>
      <Text
        strong
        className={css.guestTableTitle}
        style={{ marginBottom }}
```